### PR TITLE
chore(skip-release): prepare 1.4.0

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Review by sbouchet — APPROVED** [review]
LGTM

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -10,6 +10,13 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>1.4.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/553">Describe pod(s) like like kubectl</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:15]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/553">Provide Describe pod(s) action (on the lines of kubectl)</a></li>
```

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -10,6 +10,13 @@
...
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/757">Toolkit window stays empty if current cluster isn't reachable</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:16]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/757">Fix: Toolkit window stays empty if the current cluster isn't reachable</a></li>
```

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -10,6 +10,13 @@
...
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/755">Cannot create or update resource with managed fields</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:17]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/755">Fix: Cannot create or update resource with managed fields</a></li>
```

File: src/main/resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -10,6 +10,13 @@
...
+        </ul>
+
         <h3>1.3.0</h3>
         <ul>
             <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/726">Support for clusters using OpenID auth providers</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:22]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/726">Provide support for clusters using OpenID auth providers</a></li>
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
updated wordings

**Review by mohitsuman — APPROVED** [review]
lgtm

**Comment by adietish** [comment]
version 1.4.0.4 submitted to marketplace for approval
image

**Pull request merged**